tests/test_request.py

```diff
@@ -128,3 +123,87 @@ def test_to_dict(self, sample_request_with_user: Request) -> None:
             "uri_params": {},
             "user": f"User username={sample_request_with_user.user.username}",  # type: ignore
         }
+
+
+class TestWebSocketRequest:
+    @pytest.mark.parametrize(
+        "action_type, is_connection_req",
+        [
+            (ActionType.CONNECT, True),
+            (ActionType.DISCONNECT, False),
+            (ActionType.MESSAGE, False),
+        ],
+    )
+    def test__is_connection_request__checks_if_request_was_a_connection_request(
+        self, action_type: str, is_connection_req: bool
+    ) -> None:
+        request = WebSocketRequest(
+            "",
```

Comment:
Please, use the keywords arguments, otherwise, it is hard to see what you want to pass here.